src/rgw/driver/rados/rgw_rados.h

```diff
@@ -876,16 +877,18 @@ class RGWRados
         State() : completion(NULL), ret(0) {}
       } state;
 
-
       explicit Stat(RGWRados::Object *_source) : source(_source) {}
 
       int stat_async(const DoutPrefixProvider *dpp);
       int wait(const DoutPrefixProvider *dpp);
+#if 0 // TODO: implement
       int stat();
+#endif
+
```

Comment:
you can just remove the line instead of `#if`'ing it out. someday we might consider deprecating and removing the old orphan stuff, along with this `Stat` class